runtime/lua/provider/health.lua

```diff
@@ -0,0 +1,55 @@
+local M = {}
+
+-- Returns true if `cmd` exits with success, else false.
+local function cmd_ok(cmd)
+  vim.fn.system(cmd)
+  return vim.v.shell_error == 0
+end
+
+local function executable(exe)
+  return vim.fn.executable(exe) == 1
+end
+
+local function empty(s)
+  return vim.trim(s) == ''
+end
+
+local function clipboard()
+  vim.health.report_start('Clipboard (optional)')
+
+  if
+    os.getenv('TMUX')
+    and executable('tmux')
+    and executable('pbpaste')
+    and not cmd_ok('pbpaste')
+  then
+    local tmux_version = vim.fn.matchstr(vim.fn.system('tmux -V'), [[\d\+\.\d\+]])
```

Comment:
This seems to be equivalent to `string.match(vim.fn.system('tmux -V'), '%d+%.%d+')`.

Reply:
You're right. Thanks!